src/saga/adaptors/torque/torquejob.py

```diff
@@ -867,17 +821,15 @@ def _job_get_info(self, job_id, reconnect):
                     val = val.strip()
 
                     # The ubiquitous job state
-                    if key in ['job_state']: # PBS Pro and TORQUE
-                        job_info['state'] = _torque_to_saga_jobstate(val)
+                    if key in ['job_state']:
+                        job_state = val
 
```

Comment:
Yeah, I guess that would also work, and would leave the interpretation localized...